activestorage/lib/tasks/activestorage.rake

```diff
@@ -12,4 +12,15 @@ namespace :active_storage do
       Rake::Task["app:active_storage:install:migrations"].invoke
     end
   end
+
+  # desc "Copy over the migrations needed to the application upgrading"
+  task update: :environment do
```

Comment:
Please let me know what do you think about this approach?

Since we decided edit the existing migration "create_active_storage_tables" in #33405 in order to generate only one migration for a new app I think we should consider merging this PR - https://github.com/rails/rails/pull/32042 (It would be strange that Rails 6.0 generates migration with version 5.2).

/cc @georgeclaghorn, @matthewd